package.json

```diff
@@ -17,6 +17,7 @@
   },
   "scripts": {
     "dev": "grunt monitor",
+    "build": "grunt release",
```

Comment:
Should be probably called `release` instead of `build`.

Reply:
@DanielRuf 
> Please revert the version changes in package.json.

It's embarassing... 😵 Travis CI told me also.

> Should be probably called release instead of build.

OK, either one is fine. Fixed!